modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -182,6 +182,24 @@ supports up to `64` threads. If this field is empty, the Ingress Controller uses
 * `tlsInspectDelay` specifies how long the router can hold data to find a matching route. Setting this value too short can cause the router to fall back to the default certificate for edge-terminated or reencrypted routes, even when using a better matched certificate. If unset, the default inspect delay is `5s`.
 
 * `tunnelTimeout` specifies how long a tunnel connection, including websockets, remains open while the tunnel is idle. If unset, the default timeout is `1h`.
+
+|`logEmptyRequests`
+|`logEmptyRequests` specifies how connections on which no request is received and logged. These empty requests come from load balancer health probes or web browser speculative connections, preconnect, and logging these requests can be undesirable. However, these requests can be caused by network errors, in which case logging empty requests can be useful for diagnosing the errors. These requests can be caused by port scans, and logging empty requests can aid in detecting intrusion attempts. Allowed values for this field are `Log` and `Ignore`. The default value is `Log`.
```

Comment:
- I don't understand the first sentence because "specifies how connections on which" is unclear and also like it is missing an object? If correct, I'd consider something like: "`logEmptyRequests` specifies connections for which no request is received and logged."
- The second sentence seems like a run-on? Not sure what "preconnect" is referring to here; is that another name for web browser speculative connections? If so, you could try saying "known as preconnect" or something. Consider also making the last part its own sentence: "Logging empty requests can be undesirable." (Also might help to suggest why it is undesirable.)